packages/orchestrator-ui-components/src/api/index.ts

```diff
@@ -24,9 +24,9 @@ import { ProductDefinition } from '@/types';
 
 import { getAxiosInstance } from './axios';
 
-const CIM_FORMS_ENDPOINT = 'surf/cim/forms/';
 const PROCESS_ENDPOINT = 'processes/';
 const PRODUCTS_ENDPOINT = 'products/';
+const FORMS_ENDPOINT = 'surf/forms/';
```

Comment:
This is Surf specific no? Is there no way to get this in the gitlab version? For example by import the BaseClient and extending it there?